auth/jwt/transport.go

```diff
@@ -0,0 +1,89 @@
+package jwt
+
+import (
+	"fmt"
+	stdhttp "net/http"
+	"strings"
+
+	"github.com/go-kit/kit/transport/grpc"
+	"github.com/go-kit/kit/transport/http"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	BEARER        string = "bearer"
+	BEARER_FORMAT string = "Bearer %s"
+)
+
+// moves JWT token from request header to context
+// particularly useful for servers
+func ToHTTPContext() http.RequestFunc {
+	return func(ctx context.Context, r *stdhttp.Request) context.Context {
+		token, ok := extractTokenFromAuthHeader(r.Header.Get("Authorization"))
+		if !ok {
+			return ctx
+		}
+
+		return context.WithValue(ctx, JWTTokenContextKey, token)
+	}
+}
+
+// moves JWT token from context to request header
+// particularly useful for clients
+func FromHTTPContext() http.RequestFunc {
```

Comment:
Likewise above re: comment. Further comments about function comments in this file elided.